css-overflow-4/Overview.bs

```diff
@@ -1048,6 +1059,15 @@ Fragmentation of Overflow: the 'continue' property</h3>
 			Breaks applying to other [=fragmentation contexts=]
 			(such as pagination of this box itself)
 			do not cause any content to be discarded.
+
+		<dt><dfn>collapse</dfn>
+		<dd>
+			If the box is a [=block container=], then it must
+			[=establish an independent formatting context=] that also becomes a [=line-clamp container=].
+			This causes all content after the [=clamp point=] to be visually hidden, and to not be
+			taken into account for the box's [=automatic block size=].
+
+			This keyword has no effect on [=multicol containers=].
```

Comment:
Couldn't we make it work on multicols too? Is there any particular difficulty in inserting a clamp point after N lines or after the last line of the last non-overflow column, and hiding overflow columns in any case?

Reply:
In inserting a clamp point after N lines, no. For adding it in the last line of the last non-overflowing column, at least in my implementation in Chromium it might not be easy, because the machinery to figure out whether a line is the last line that doesn't overflow, including accounting for border/padding/margin, would have to interact with the fragmentation machinery.